incubator/hnc/README.md

```diff
@@ -129,3 +129,12 @@ milestones are defined:
 
 Non-coding tasks are also tracked in the [HNC
 project](https://github.com/kubernetes-sigs/multi-tenancy/projects/4).
+
+## Releasing
+
+[comment]: # (TODO: pull the code directly from Github)
+
+Check out the code to the version that you want to release and invoke
+`make release` in `incubator/hnc`. It will use Cloud Build (on your GCP account)
+to build the container image remotely while building YAMLs locally (in `manifests/hnc-manager.yaml`).
+The container image will be upload to MTWG's Docker Hub and release YAMLs should be hosted on Github.
```

Comment:
Can you explain this a bit more? At the end of the Cloud Build process, the image is in GCR, not Docker Hub. Can you add the _specific_ instructions about how to get it to the right place? Eg, something like:

- Make sure you have write access to <docker hub ID>
- Pull the image via `docker pull gcr.io/YOUR_PROJECT_ID/etc`
- Push the image via `docker push ETC`

You should basically have instructions I can cut-and-paste.